File: examples/php/constructs/string_equality.php
<?php
// Ported from examples/lumen/constructs/string_equality.lm by scripts/port_examples.py; edit the Lumen original, not this file.
$x = "hello";
$y = "hello";
$z = "world";
print(($x == $y) . "\n");
print(($x == $z) . "\n");
print(($x != $z) . "\n");

File: examples/rust/constructs/string_equality.rs
// Ported from examples/lumen/constructs/string_equality.lm by scripts/port_examples.py; edit the Lumen original, not this file.
fn main() {
    let x = "hello";
    let y = "hello";
    let z = "world";
    println!("{}", x == y);
    println!("{}", x == z);
    println!("{}", x != z);
}
